lib/ClangImporter/ImportType.cpp

```diff
@@ -2431,6 +2431,23 @@ DefaultArgumentKind ClangImporter::Implementation::inferDefaultArgument(
           return DefaultArgumentKind::EmptyArray;
       }
     }
+  } else if (const clang::TypedefType *typedefType =
+                 type->getAs<clang::TypedefType>()) {
+
+    // Get the AvailabilityAttr that would be set from CF/NS_OPTIONS
+    const clang::AvailabilityAttr *availabilityAttr =
+        typedefType->getDecl()->getAttr<clang::AvailabilityAttr>();
+
+    if (availabilityAttr && availabilityAttr->getUnavailable()) {
```

Comment:
Please use `isUnavailableInSwift` instead.

Reply:
Done. Should I leave enableObjCInterop to true?